AmericanMedicalAssociation/AMA-style-guide
# [EWL-4193] SG | Design State Data Viewer

## Ticket(s)

**Jira Ticket**

- [EWL-4193: SG | Design State Data Viewer](https://issues.ama-assn.org/browse/EWL-4193)


## Description

Adds a State Data Viewer template and State Data Info organism.

## To Test

Templates > State Data Viewer: see that the pattern exists and verify that it looks appropriate based on the wireframes in the ticket
Organisms > Sections > State Data Info: see that this pattern exists


## Relevant Screenshots/GIFs

![4193-1](https://user-images.githubusercontent.com/12160398/32680577-f3425a80-c630-11e7-9ebf-ff5c9c8a4012.png)

## Remaining Tasks

- [ ] Is it necessary for `state-data-info` to have its own .json? I left this out for time's sake, so let me know if I need to add it.
- [ ] Suggestions for more efficient use of `{% set content %}` in `state-data.twig` welcome.
- [ ] Didn't have a better image available to use - if the landscape-3x2 element should be replaced with something else (and/or the content placeholder image) let me know
- [ ] This pattern uses a deprecated layout pattern, `layout-one-up-center`. If I need to refactor to not use this, let me know.

## Additional Notes

N/A
